arbitrum-docs/launch-orbit-chain/how-tos/orbit-sdk-deploying-token-bridge.md

```diff
@@ -24,30 +26,36 @@ See the [`ERC-20` token bridge overview](/build-decentralized-apps/token-bridgin
 
 :::
 
-### Token Bridge Deployment Steps
+### Token bridge deployment steps
 
-Following the [deployment](orbit-sdk-deploying-rollup-chain.md) and initialization of the Orbit chain, the subsequent phase involves deploying contracts on both the parent and child chains.
+Once an Orbit chain has been deployed and initialized, the bridge contracts need to be deployed on both the parent and child chains. This process involves several steps:
 
-To establish and configure the token bridge effectively, the process can be broken down into the following steps:
-
-1. **[Token Approval](#step-1)**
-2. **[ Token Bridge Contract Deployment ](#step-2)**
-3. **[ Transaction Recipient and Checking for Deployment on Child Chain ](#step-3)**
-4. **[ Deployment Information and Contract Addresses ](#step-4)**
-5. **[ Setting Up the WETH Gateway ](#step-5)**
+1. **[Token approval](#step-1)**
+2. **[Token bridge contract deployment](#step-2)**
+3. **[Transaction recipient and checking for deployment on child chain](#step-3)**
+4. **[Deployment information and contract addresses](#step-4)**
+5. **[Setting up the WETH gateway](#step-5)**
 
 :::info
 
-The token bridge deployment process depends on the type of Orbit chain. In the following steps, the main flow is the same for all different kinds of Orbit chains except [step 1](#step-1), which is only needed for **Custom fee token** Orbit chains, and [step 5](#step-5), just for `ETH`-based Orbit chains.
+The token bridge deployment process is the same for all Orbit chains types except for the following:
+- **Custom fee token Orbit chains** which require [token approval](#step-1).
+- **`ETH`-based Orbit chains**, for which you need to [set up a WETH gateway](#step-5).
 
 :::
 
-### 1. Token Approval (Custom fee token Orbit chains only){#step-1}
+### 1. Token approval {#step-1}
```

Comment:
Curious - is there any reason we can't remove these unconventional `{}` notations and just link to the headers as-usual, where needed?